vpr/src/route/connection_router.cpp

```diff
@@ -515,14 +551,17 @@ void ConnectionRouter<Heap>::timing_driven_add_to_heap(const t_conn_cost_params
     float new_total_cost = next.cost;
     float new_back_cost = next.backward_path_cost;
 
-    if (new_total_cost < best_total_cost && new_back_cost < best_back_cost) {
+    if (new_total_cost < best_total_cost && ((new_back_cost < best_back_cost) || (rcv_path_manager.is_enabled()))) {
```

Comment:
Should reverse the order of is_enabled check and cost comparisions?

Reply:
Is this just a branch predictor optimization? I changed it just curious

Reply:
In C++ a || b implies "execute statement a, if a is false, execute b".  In the case RCV is true, you want to not execute the float comparision, and in the case RCV is false, it is very cheap to check if a bool is set or not.   It might even be free if the is_enabled_ variable is in a register.